src/main/java/org/opensearch/security/transport/SecurityInterceptor.java

```diff
@@ -408,4 +413,19 @@ public String executor() {
         }
     }
 
+    private boolean shouldUseJDKSerialization(Connection connection) {
+        if(connection.getVersion().before(ConfigConstants.FIRST_CUSTOM_SERIALIZATION_SUPPORTED_OS_VERSION)) {
+            return true;
+        } else {
+            String customSerializationEnabledNodeAttr = connection.getNode().getAttributes().get(
```

Comment:
Can `getAttributes()` ever return `null`?

Reply:
We've removed the usage of node property, so this is no longer applicable.